api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/AbstractReferenceHandlingTranslator.java

```diff
@@ -109,6 +110,10 @@ protected Reference createPractitionerReference(@NotNull Provider provider) {
 		return reference;
 	}
 	
+	protected Reference createOrderReference(@NotNull Order order) {
+		return new Reference().setReference(FhirConstants.ORDER + "/" + order.getUuid()).setType(FhirConstants.ORDER);
```

Comment:
This should still be "ServiceRequest" since "Order" isn't a valid FHIR object type.

Reply:
In this method, it may make sense to differentiate `DrugOrder`s, but in that case have them return a reference to a "MedicationRequest". However, that only makes sense once the `MedicationRequest` resource is created.